src/views/common/TimeScrollView/TimeScrollViewInteractions/useTimeScrollZoom.ts

```diff
@@ -0,0 +1,62 @@
+import { defaultZoomScaleFactor, ZoomDirection } from 'contexts/RecordingSelectionContext';
```

Comment:
Again, this file was meant to better encapsulate a small area of functionality. I also moved the throttling to the standard framework.